client/src/pages/CalendarPage/CalendarPage.module.css

```diff
@@ -0,0 +1,53 @@
+.bg {
+  background-image: linear-gradient(
+    #032567 30%,
+    rgba(195, 221, 253, 30) 60%,
+    #032a5c
+  );
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.calendarHeader {
+  color: white;
+  font-weight: 600;
+  font-family: "Source Sans Pro", sans-serif;
```

Comment:
I think we might be able to apply the `font-family` to the actual `html` or `body` class (below) such that we don't need to constantly type in `font-family` all the time. (Then for other styles that might require different font families, we can override it with our additional CSS modules)

Reply:
Since this would require a non-trivial amount of refactoring, I will do this in a separate PR or the next PR.

Reply:
Sounds good. Noted.